README - INSTALL INSTRUCTIONS/QBOX/README.md

```diff
@@ -310,12 +343,50 @@ ApartmentOptions = {
         enter = vec3(-614.58, 46.52, 43.59)
     },
 }
+ ```
+# Running The SQL
+## If You Have Previously Had qbx_properties You Will Need To Drop properties and properties_decorations, If You Are A New Server Ignore This Specific Line
+### Step 1: Open Your Preferred Database Management System (PhpMyAdmin, HeidiSQL, etc.. )
+### Step 2: Run The SQL
+
+### If you get an error for foreign Key  Restraints You Didn't Drop The Tables
+
+# Updating Your server.cfg
```

Comment:
All these headers seem a bit extensive. The meaning of a header looses its value.